test/parallel/test-util-emit-experimental-warning.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+// Flags: --expose-internals
+const common = require('../common');
+const assert = require('assert');
+const { emitExperimentalWarning } = require('internal/util');
+
+// This test ensures that the emitExperimentalWarning in internal/util emits a
+// warning when passed a unsupported feature and that it simply returns
```

Comment:
Nit: :%s/a/an ?

Reply:
@lpinca : Good catch. I've updated it. Thanks !